src/main/java/org/betonquest/betonquest/conversation/SlowTellrawConvIO.java

```diff
@@ -28,7 +30,12 @@ public SlowTellrawConvIO(final Conversation conv, final OnlineProfile onlineProf
             string.append(color);
         }
         this.npcTextColor = string.toString();
-
+        int delay = BetonQuest.getInstance().getPluginConfig().getInt("conversation_IO_config.slowtellraw.message_delay", 10);
+        if (delay <= 0) {
+            BetonQuest.getInstance().getLogger().warning("Invalid message delay for SlowTellraw Conversation IO, using default value of 10");
+            delay = 2;
```

Comment:
Actually set the delay to 10 as the warning says! Additionally, actually telling what the invalid value is could be helpful as well.

```suggestion
            BetonQuest.getInstance().getLogger().warning("Invalid message delay of " + delay + " for SlowTellraw Conversation IO, using default value of 10 ticks");
            delay = 10;
```